manifests/mod/lbmethod_bybusyness.pp

```diff
@@ -0,0 +1,19 @@
+# @summary
+#   Installs `lbmethod_bybusyness`.
+# 
+# @param apache_version
+#   Version of Apache to install module on.
+# 
+# @see https://httpd.apache.org/docs/2.4/mod/mod_lbmethod_bybusyness.html for additional documentation.
+#
+class apache::mod::lbmethod_bybusyness (
+  Optional[String] $apache_version   = $apache::apache_version,
+) {
+  require apache::mod::proxy_balancer
+
+  if versioncmp($apache_version, '2.3') >= 0 {
+    ::apache::mod { 'lbmethod_bybusyness': }
```

Comment:
This is true for all files, but it's no longer needed to prefix with `::`
```suggestion
    apache::mod { 'lbmethod_bybusyness': }
```

Reply:
removed unnecessary prefix